### ISSUE TITLE
Cache some ESI responses which should change infrequently or never

### ISSUE THREAD

**Issue by SoCariEVE** [MonoliYoda/evve_dashboard_dev#6]
Looking at [views.py](https://github.com/MonoliYoda/evve_dashboard_dev/blob/master/dashboard/views.py) (in particular the `ajax_get_timers()`, `ajax_new_timer()` and `ajax_remove_timer()` functions) there look to be some cases where relatively immutable data like a character's `corporation_id` is repeatedly requested from ESI.

We should probably move that to a single call whenever the user changes character, and cache it.

**Comment by MonoliYoda** [comment]
Yep, I was wondering what the best way to go about it is. It's only a single call per each view, so not a big deal, but there's definitely room for optimization.
